### DISCUSSION TITLE
The encoding delay of converting multi-frame images to webp is very long

### DISCUSSION THREAD

**Discussion by zhouleiMax** [ImageMagick/ImageMagick#6415] [Help]
### ImageMagick version

7.1.0-52

### Operating system, version and so on

Linux VM-232-130-centos 3.10.107-1-tlinux2_kvm_guest-0054

### Description

In the coder/webp.c file, 
in the WriteAnimatedWEBPImage() function，
What is the function of WebPEncode() function, This function takes a lot of time.
The WebPAnimEncoderAdd() function will be used later, why use the WebPEncode() function.Both functions have the ability to compress pixel data.
thank you very much !

**Comment by dlemstra** [comment]
It does look like we don't need to call `WebPEncode`. I will investigate this and see if we can remove that call.

**Comment by zhouleiMax** [comment]


**Reply by dlemstra** [reply]
Please don't use our repository to ping other people.
